Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/approximated_centroidal_Voronoi_diagram_remeshing.h

```diff
@@ -1390,6 +1390,9 @@ acvd_impl(TriangleMesh& tmesh,
 *
 * \cgalNamedParamsEnd
 *
+*
+* @pre tmesh is a triangulated surface mesh and has exactly one connected component.
```

Comment:
`tmesh` -> `` `tmesh` ``?
(see also text in Surface_mesh_skeletonization/include/CGAL/extract_mean_curvature_flow_skeleton.h)